keyboards/doio/kb16/rev1/info.json

```
@@ -1,56 +1,15 @@
 {
-    "keyboard_name": "KB16-01",
-    "manufacturer": "DOIO",
-    "url": "",
-    "maintainer": "HorrorTroll",
     "usb": {
-        "vid": "0xD010",
-        "pid": "0x1601",
-        "device_version": "0.0.1",
-        "force_nkro": true
+        "device_version": "0.0.1"
     },
-    "rgb_matrix": {
-        "animations": {
-            "alphas_mods": true,
-            "gradient_up_down": true,
-            "breathing": true,
-            "band_sat": true,
-            "band_val": true,
-            "band_pinwheel_sat": true,
-            "band_pinwheel_val": true,
-            "cycle_all": true,
-            "cycle_up_down": true,
-            "cycle_out_in": true,
-            "cycle_pinwheel": true,
-            "dual_beacon": true,
-            "rainbow_pinwheels": true,
-            "raindrops": true,
-            "jellybean_raindrops": true,
-            "hue_breathing": true,
-            "hue_pendulum": true,
-            "hue_wave": true,
-            "pixel_rain": true,
-            "pixel_flow": true,
-            "pixel_fractal": true,
-            "solid_reactive_simple": true,
-            "solid_reactive": true,
-            "solid_reactive_multiwide": true,
-            "solid_reactive_multicross": true,
-            "solid_reactive_multinexus": true,
-            "multisplash": true,
-            "solid_multisplash": true
-        },
-        "default": {
-            "animation": "cycle_up_down"
-        },
-        "driver": "ws2812",
-        "max_brightness": 200
+    "features": {
+        "space_cadet": false,
```

Could probably disable grave escape, too.
```suggestion
        "grave_esc": false,
        "space_cadet": false,
```